ngraph/core/include/ngraph/op/parameter.hpp

```diff
@@ -41,7 +41,6 @@ namespace ngraph
                 void set_is_relevant_to_shapes(bool is_relevant);
 
                 const PartialShape& get_partial_shape() const { return m_partial_shape; }
-                PartialShape& get_partial_shape() { return m_partial_shape; }
```

Comment:
Just a question to make sure - this is a removal of one of the API methods of the Parameter class. The situation is a little more tricky in this case though because just one overload gets removed so it's still possible to retrieve the partial shape for Parameter objects. @ilyachur does that look ok from your perspective?

Reply:
Honestly it is a backward incompatible change.